src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,  initial-scale=1.0, maximum-scale=1.0, minimum-scale=1. 0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+  </head>
+  <body>
+    <div class="card card--main" data-qa="card">
+      <div class="card__pic_container">
+        <img
+          class="card__pic" 
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K   
+          Monoblock"
+        >
+      </div>
+      <div class="card__about card__about--font">
+        APPLE A1419 iMac 27" Retina 5K   
+        Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__prod_id card__prod_id--font">
+        Product code: 195434
+      </div>
+      <div class="stars">
+        <ul class="stars__average stars__average--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <div class="stars__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__cost card__cost--font">
+        <div>Price:</div>
+        <div class="card__cost_value">$2,199</div>
+      </div>
+      <a 
+        href="#"
+        class="card__link_buy"
+        data-qa="hover"
+      >
+        <div class="card__button card__button--font">
+          <div class="card__button_text">Buy</div>
+        </div>
+      </a>
+    </div>
```

Comment:
```suggestion
      <div class="stars">
        <ul class="stars__average stars__average--4">
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
        </ul>
        <div class="stars__reviews">Reviews: 5</div>
      </div>
      
      <div class="card__cost card__cost--font">
        <div>Price:</div>
        <div class="card__cost_value">$2,199</div>
      </div>
      
      <a 
        href="#"
        class="card__link_buy"
        data-qa="hover"
      >
        <div class="card__button card__button--font">
          <div class="card__button_text">Buy</div>
        </div>
      </a>
    </div>
```

Reply:
i fixed it, but... Could you explain to me why the checklist has spaces between <li> ? 
![image](https://user-images.githubusercontent.com/70288388/201530361-6e87f80c-4e9a-4053-8f85-095a5f0803a6.png)

Reply:
In example from checklist we have tag `a` inside `li`, but in our case all the `li`s are empty so there is no need to provide new lines between them. You can see that the second variant looks pretty clean and easy-reading without new lines.
![image](https://user-images.githubusercontent.com/96409563/201636575-fb7d8093-2086-4fe5-a1ca-ea5ee18aa908.png)
![image](https://user-images.githubusercontent.com/96409563/201636609-bb97955b-a6b6-4068-a5e8-d8977b80a1e0.png)